### PR TITLE
Added Ijanki bundles

### PR DESCRIPTION
**Pull request by iJanki**
| Q             | A
| ------------- | ---
| License       | MIT

### CODE REVIEW

**Comment by iJanki** [comment]
@fabpot What's the waiting time to get reviewed?

File: ijanki/prometheus-bundle/0.9/config/routes/ijanki_prometheus.yaml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+ijanki_prometheus:
+    resource: "@IjankiPrometheusBundle/Resources/config/routing.yml"
+    prefix: /
```
Comment (Nyholm) [inline ijanki/prometheus-bundle/0.9/config/routes/ijanki_prometheus.yaml]: Is this needed? Either add a value that changes the default behavior or remove it.

File: ijanki/prometheus-bundle/0.9/manifest.json [hunk 2]
```diff
@@ -0,0 +1,10 @@
+{
+    "bundles": {
+        "Ijanki\\Bundle\\PrometheusBundle\\IjankiPrometheusBundle": [
+            "all"
```
Comment (Nyholm) [inline ijanki/prometheus-bundle/0.9/manifest.json]: Lets do this on the same line.

File: ijanki/prometheus-bundle/0.9/config/packages/ijanki_prometheus.yaml [hunk 3]
```diff
@@ -0,0 +1,22 @@
+ijanki_prometheus:
+    storage_adapter_service: TweedeGolf\PrometheusClient\Storage\ApcuAdapter
+    metrics_path: /metrics
+    collectors:
+        http_requests_total:
```
Comment (Nyholm) [inline ijanki/prometheus-bundle/0.9/config/packages/ijanki_prometheus.yaml]: You do not need all the default config here. Put a link to your documentation instead.

Comment (iJanki) [reply ijanki/prometheus-bundle/0.9/config/packages/ijanki_prometheus.yaml]: The point of this opinionated bundle is to provide these default metrics at installation time.

Comment (Nyholm) [reply ijanki/prometheus-bundle/0.9/config/packages/ijanki_prometheus.yaml]: Sorry, I misread the code. You are actually creating collectors that you name and configure. Removing `http_requests_total`,  `http_request_duration_seconds` etc would mean we are removing services.

These are fine to leave here :+1:

Make sure to address my other comments and I'll be happy to merge.

File: ijanki/prometheus-bundle/0.9/config/routes/ijanki_prometheus.yaml (hunk 1 above)
Comment (iJanki) [reply ijanki/prometheus-bundle/0.9/config/routes/ijanki_prometheus.yaml]: So the routing will be automatically loaded by default? Ok, I removed it.

File: ijanki/prometheus-bundle/0.9/manifest.json (hunk 2 above)
Comment (iJanki) [reply ijanki/prometheus-bundle/0.9/manifest.json]: Ok, i put on one line, thanks.

File: ijanki/prometheus-bundle/0.9/config/packages/ijanki_prometheus.yaml (hunk 3 above)
Comment (iJanki) [reply ijanki/prometheus-bundle/0.9/config/packages/ijanki_prometheus.yaml]: Ok i should have resolved the issues, please review.

**Comment by iJanki** [comment]
@Nyholm, all good now?

**Pull request merged**